code/modules/mob/living/simple_animal/abnormality/waw/silence.dm

```diff
@@ -0,0 +1,66 @@
+//Coded by me, Kirie Saito!
+/mob/living/simple_animal/hostile/abnormality/silence
+	name = "The Price of Silence"
+	desc = "A scythe with a clock attached, quitely ticking."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "silence"
+	threat_level = WAW_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(0, 0, 30, 40, 40),
+		ABNORMALITY_WORK_INSIGHT = 0,
+		ABNORMALITY_WORK_ATTACHMENT = list(0, 0, 10, 20, 30),
+		ABNORMALITY_WORK_REPRESSION = list(0, 0, 50, 45, 45)
+			)
+	start_qliphoth = 1
+	work_damage_amount = 10
+	work_damage_type = PALE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/thirteen,
+		/datum/ego_datum/armor/thirteen,
+		)
+//	gift_type = /datum/ego_gifts/thirteen
+
+	var/meltdown_cooldown_time = 780 SECONDS	//13 minutes
+	var/meltdown_cooldown
+	var/worldwide_damage = 70	//If you're unarmored, it obliterates you
+	var/safe = FALSE //work on it and you're safe for 13 minutes
+	var/reset_time = 180 SECONDS //Don't hit everyone with the global pale if it was hit in a small period of time
+	var/ready = TRUE
+
+/mob/living/simple_animal/hostile/abnormality/silence/Initialize()
+	..()
+	meltdown_cooldown = world.time + meltdown_cooldown_time
+
+/mob/living/simple_animal/hostile/abnormality/silence/Life()
+	. = ..()
+	if(meltdown_cooldown < world.time)
+		meltdown_cooldown = world.time + meltdown_cooldown_time
+		sound_to_playing_players('sound/abnormalities/silence/bong.ogg')
+		if(!safe)
+			datum_reference.qliphoth_change(-1)
+			sound_to_playing_players('sound/abnormalities/silence/bong.ogg')	//Church bells ringing, whether it happens or not.
+		safe = FALSE
+
+/mob/living/simple_animal/hostile/abnormality/silence/success_effect(mob/living/carbon/human/user, work_type, pe)
+	if(!safe)
+		to_chat(user, "<span class='nicegreen'>It does not toll for thee, Not yet.</span>")
+	safe = TRUE
+	return
+
+//Meltdown
+/mob/living/simple_animal/hostile/abnormality/silence/zero_qliphoth(mob/living/carbon/human/user)
+	datum_reference.qliphoth_change(1)
+	if(!ready)
+		return
+	for(var/mob/living/L in GLOB.player_list)
```

Comment:
I think it'd be cool if this had a stoplag proc in it so that it didn't just hit everyone at once, but ding through them one at a time.

Reply:
Added!

Reply:
Where?